ssl/statem/statem_clnt.c

```diff
@@ -2927,6 +2927,7 @@ static int tls_construct_cke_dhe(SSL *s, WPACKET *pkt)
     if (encoded_pub_len == 0) {
         SSLfatal(s, SSL_AD_INTERNAL_ERROR, ERR_R_INTERNAL_ERROR);
         EVP_PKEY_free(skey);
```

Comment:
This call is suspicious, is this just a typo error where skey is meant to be ckey here? @mattcaswell ?